### PR TITLE
Update what is new onmetal 

### PR DESCRIPTION
**Pull request by aurimasmick**
Update flavor table to onmetal v2

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3bfec9369e/how-to/). :bow:

File: content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md [hunk 1]
```diff
@@ -54,17 +51,13 @@ configurations:
...
+| **OnMetal I/O v2**  | **Dual 2.6 GHz, 10 core Intel® Xeon® E5-2660 v3**, 128GB RAM, 240GB hot-swappable SSDs configured in a RAID 1 mirror, **Dual Seagate Nytro XP6302 1.6 TB PCIe flash cards**,  redundant 10Gb/s network  | Designed to support low-latency and extreme throughput to local storage. Optimized for large relational databases and NoSQL data stores like Cassandra or MongoDB, latency-sensitive applications, online transaction processing and real-time analytics.   |
+
+All OnMetal v2 are built on second generation Open Compute hardware, General Purpose v2 utilise Intel Xeon E5-2620 v3 processors and I/O v2 - Intel Xeon E5-2660 v3 processor. Fast and redundant 10-gigabit network is used for all instance types, because
+network performance is becoming increasingly important. Integration with Cloud Networks and RackConnect 3.0 allow to isolate traffic for complete security. All OnMetal v2 server flavors have redundant 800GB hot-swappable SSDs configured in a RAID 1 mirror to reduce the risk of lost data. The  I/O v2 flavor also has two Seagate Nytro XP6302 1.6 TB PCIe flash cards for extreme throughput to local storage. Cloud
```
Comment (CrashenX) [inline content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md]: @jayofdoom has a training slide that says v2 vs v3 for processors. However, https://one.rackspace.com/display/teeth/OnMetal+v2+Flavors and https://www.rackspace.com/cloud/servers/onmetal/specs show v3 which matches this PR. Might want to verify with @jayofdoom , but lgtm

File: content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md [hunk 2]
```diff
@@ -106,12 +99,6 @@ information on generating SSH keys, see [Manage SSH Key Pairs for Cloud Servers
 
 ### Limitations
 
-After an OnMetal server is built, there is a slight delay in network
-availability. Even after the status reads ACTIVE, a few additional
-minutes (3 to 4) are needed for the network configuration to complete.
-After the new OnMetal server pings successfully, you can begin to use
-it.
-
```
Comment (CrashenX) [inline content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md]: Why was this removed? I'm not sure what the delay is, but this is still true. You can science it by doing a few boots.

File: content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md [hunk 3]
```diff
@@ -54,17 +51,13 @@ configurations:
...
+| **OnMetal GP v2 Large**  | **Dual 2.4 Ghz, 6 core Intel® Xeon® E5-2620 v3**, 128GB RAM, redundant 800GB hot-swappable SSDs configured in a RAID 1 mirror, redundant 10Gb/s network  | Optimized for memory-heavy workloads. Especially useful for high RAM activity like in-memory SQL configurations, caching servers, and search indexes.   |
+| **OnMetal I/O v2**  | **Dual 2.6 GHz, 10 core Intel® Xeon® E5-2660 v3**, 128GB RAM, 240GB hot-swappable SSDs configured in a RAID 1 mirror, **Dual Seagate Nytro XP6302 1.6 TB PCIe flash cards**,  redundant 10Gb/s network  | Designed to support low-latency and extreme throughput to local storage. Optimized for large relational databases and NoSQL data stores like Cassandra or MongoDB, latency-sensitive applications, online transaction processing and real-time analytics.   |
+
+All OnMetal v2 are built on second generation Open Compute hardware, General Purpose v2 utilise Intel Xeon E5-2620 v3 processors and I/O v2 - Intel Xeon E5-2660 v3 processor. Fast and redundant 10-gigabit network is used for all instance types, because
```
Comment (CrashenX) [inline content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md]: s/utilise/utilize/

File: content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md (hunk 1 above)
Comment (CrashenX) [inline content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md]: IO has 240G not 800G. s/All/All General Purpose/

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ff879238f2/how-to/)

File: content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md [hunk 4]
```diff
@@ -56,8 +56,8 @@ configurations:
...
-network performance is becoming increasingly important. Integration with Cloud Networks and RackConnect 3.0 allow to isolate traffic for complete security. All OnMetal v2 server flavors have redundant 800GB hot-swappable SSDs configured in a RAID 1 mirror to reduce the risk of lost data. The  I/O v2 flavor also has two Seagate Nytro XP6302 1.6 TB PCIe flash cards for extreme throughput to local storage. Cloud
+All OnMetal v2 are built on second generation Open Compute hardware, General Purpose v2 utilize Intel Xeon E5-2620 v3 processors and I/O v2 - Intel Xeon E5-2660 v3 processor. Fast and redundant 10-gigabit network is used for all instance types, because
+network performance is becoming increasingly important. Integration with Cloud Networks and RackConnect 3.0 allow to isolate traffic for complete security. All General Purpose v2 server flavors have redundant 800GB hot-swappable SSDs configured in a RAID 1 mirror to reduce the risk of lost data. The  I/O v2 flavor also has two Seagate Nytro XP6302 1.6 TB PCIe flash cards for extreme throughput to local storage. Cloud
```
Comment (CrashenX) [inline content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md]: Hmm, perhaps a better way to state this so that it covers all flavors is: "All v2 server flavors have redundant hot-swappable SSDs..." i.e. s/General Purpose// and s/800GB//

Comment (aurimasmick) [inline content/cloud-servers/what-is-new-with-onmetal-cloud-servers.md]: yep, agreed

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-39e45dbb48/how-to/).

**Comment by CrashenX** [comment]
lgtm
